en/README.md

```diff
@@ -51,12 +53,26 @@ MAVLink 1 source files can be generated for:
 * Python (2.7+, 3.3+)
 
 
-## Prebuilt MAVLink Source Files {#prebuilt_libraries}
+The following libraries have been developed outside the project:
+
+> **Note** The MAVLink project has not validated and does not provide technical support for these projects.
+
+* Clojure 
+  - [clj-mavlink](https://github.com/WickedShell/clj-mavlink) - Clojure MAVLink Bindings. 
+    - Supports both MAVLink v1 and MAVLink v2.
+* Java 
+  - [dronefleet/mavlink](https://github.com/dronefleet/mavlink) - An *idiomatic* Java SDK/API for MAVLink. 
+    - Supports both MAVLink v1 and MAVLink v2.
```

Comment:
Pretty sure this also supports signing. Might be worth mentioning that this provides a gradle plugin for the generator (that is exciting/high value for most of the Java people)